Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_check_file_existence.yml

```diff
@@ -46,7 +47,7 @@ tasks:
       version: -1
       name: Run script file exists
       description: Initiates a new endpoint script execution to check if file exists.
-      script: '|||xdr-file-exist-script-execute'
+      script: '|||xdr-run-script-file-exists'
```

Comment:
here as well. (I guess for all XDR playbooks here)

Reply:
xdr-run-script-file-exists is the original command. I have already reverted this line.